source/Calamari.Tests/KubernetesFixtures/InstallTools.cs

```diff
@@ -218,6 +219,45 @@ public async Task InstallGCloud()
                         return GetGcloudExecutablePath(destinationDirectoryName);
                     });
             }
+            
+            if (CalamariEnvironment.IsRunningOnWindows)
+            {
+                InstallGkeAuthPlugin();
+            }
```

Comment:
This is currently limited to windows because of extra stuff needed on *nix systems & mac